src/test/java/org/broadinstitute/hellbender/utils/mcmc/DoubleState.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.utils.mcmc;
+
+import java.util.Arrays;
+
+/**
+ * Simple implementation of AbstractParameterizedState containing a single numeric parameter.
+ * Used for creating simple unit tests for MCMC samplers
+ *
+ * Created by davidben on 11/24/15.
+ */
+public class DoubleState extends AbstractParameterizedState {
+
+    public static final String PARAMETER_NAME = "x";
+
+    @Override
+    protected <S extends AbstractParameterizedState> S copy(final Class<S> stateClass) {
+        return stateClass.cast(new DoubleState(this));
+    }
+
+    private DoubleState(final DoubleState state) {
+        super(state);
+    }
+
+    public DoubleState(final double value) { super(Arrays.asList(new Parameter<Double>(PARAMETER_NAME, value))); }
```

Comment:
IntelliJ likes Collections.singletonList better. also, no need to specify Double in the angle brackets.